src/test/components/Root-history.test.js

```diff
@@ -0,0 +1,185 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+// We want to test these components in isolation and tightly control the actions
+// dispatched and avoid any side-effects.  That's why we mock this module and
+// return dummy thunk actions that return a Promise.
+// jest.mock('../../actions/receive-profile', () => ({
+//   // These mocks will get their implementation in the `setup` function.
+//   // Otherwise the implementation is wiped before the test starts.
+//   // See https://github.com/facebook/jest/issues/7573 for more info.
+//   retrieveProfileFromAddon: jest.fn(),
+//   retrieveProfileFromStore: jest.fn(),
+//   retrieveProfilesToCompare: jest.fn(),
+// }));
```

Comment:
Is this a left over?

Reply:
Yeah, I just forgot to remove, thanks. I wasn't sure why they were in other test I cribbed from, and though I may need them.